packages/cli/src/lib/Compiler.ts

```diff
@@ -5,23 +5,20 @@
 import { Web3ApiManifest } from "./Web3ApiManifest";
 import { displayPath } from "./helpers/path";
 import { step, withSpinner } from "./helpers/spinner";
+import { SchemaComposer } from "./SchemaComposer";
 
 import fs, { readFileSync } from "fs";
 import path from "path";
 import * as asc from "assemblyscript/cli/asc";
-import { Manifest, Uri, Web3ApiClient, UriRedirect } from "@web3api/client-js";
+import { Manifest } from "@web3api/client-js";
 import { bindSchema, writeDirectory } from "@web3api/schema-bind";
-import { composeSchema, ComposerOutput } from "@web3api/schema-compose";
-import { EnsPlugin } from "@web3api/ens-plugin-js";
-import { EthereumPlugin } from "@web3api/ethereum-plugin-js";
-import { IpfsPlugin } from "@web3api/ipfs-plugin-js";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
 const fsExtra = require("fs-extra");
 // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
 const toolbox = require("gluegun/toolbox");
 
-export interface CompilerConfig {
+export interface BuildConfig {
```

Comment:
Nit: I like using the class name + "Config", so viewers can immediately see what the interface is for (configuring the class via the constructor). This naming convention is used throughout the rest of the code as well. So long story short, let's rename this back to "CompilerConfig"